directory-listing/common.css

```diff
@@ -58,3 +81,22 @@ footer {
 footer a {
   color: #546e7a;
 }
+
+.even-parentdir .indexcolname a{
+  position: relative;
+  color:transparent;
+}
+.even-parentdir .indexcolname a::after{
+  content:'..';
+  position:absolute;
+  left: .75rem;
+  color:#303F9F;
+}
+
+address{
```

Comment:
You don't need all these styles here, `padding` and `text-align: center` should do fine.

Reply:
I agree about the `text-align: center`, however, I'm not quite sure how would you fix it to the bottom with just the padding